src/vs/workbench/contrib/terminal/test/node/terminalProfiles.test.ts

```diff
@@ -9,7 +9,13 @@ import { detectAvailableProfiles, IStatProvider } from 'vs/workbench/contrib/ter
 
 export interface ITestTerminalConfig {
 	profiles: {
-		windows: {
+		windows?: {
+			[key: string]: { path?: string[], source?: string }
+		},
+		linux?: {
+			[key: string]: { path?: string[], source?: string }
+		},
+		osx?: {
 			[key: string]: { path?: string[], source?: string }
```

Comment:
Should this pull in the profile interface used elsewhere?

Reply:
yep, good point. now the test is failing though and i'm not sure why. wil linvestigate later per that issue